_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,366 @@
+- Improve the **quality and security** of code through continuous automated testing and external contributions (by working in the open);
+- Ensure compliance with **Information Management** policies with lifecycle management;
+- Attract IT **talent**;
+- Reduce vendor **lock-in** by increasing capacity to support source code; and
+- Obtain metrics/statistics for **reporting** and continuous improvement.
+
+The Architecture Review Committee (ARC) already endorsed the usage of 3 version control systems in [July 2019 (internal presentation)](https://dialogue/proj/ITCD-RMTI/SAS-SAS/Documents/Services/Projects/GC%20Code/ARC%20Presentation%20on%20GCCode%20Governance.pptx):
+
+- [GitHub](https://github.com): Share with public (Unclassified);
+- [GCcode (internal)](https://gccode.ssc-spc.gc.ca): Share with GC (up to Protected A); and
+- [Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/): Share within ESDC (up to Protected B).
+
+However, work remains in some areas to drive adoption.
+For instance, GCcode is not officially supported by SSC and lack some modern DevOps features (such as deployment to external cloud providers).
+This strategy’s goal is to define what the target state of the version control systems should be, and provide a roadmap in getting to this target state.
+
+This strategy complements existing IT initiatives, such as [ESDC IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), and supports them with new activities (see [Coherent set of actions](#coherent-set-of-actions)).
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) regarding usage of the Version Control Systems.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see [Coherent set of actions](#coherent-set-of-actions)).
+
+This policy becomes active when IT Products are to be built.
+Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
+
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+- ESDC Information Management Policy (being drafted)
+- ESDC Security Policy (being drafted)
+
+### Development and Operations Teams (incl. SSC) <!-- omit in toc -->
+
+1. Use a DevOps platform, that includes a Git-based Version Control System (VCS), among the following options, in that order of preference:
```

Comment:
You will be asked:
- what about mainframes and our COBOL code? Do they have a fit?
- poorly coded code, inherited from decades of technical debt that includes sensitive information (e.g. IP addresses, passwords). We're talking hundres of thousands lines of codes here. I was told by BSI they more than once got ATIPed on their code and had to take efforts in stripping the sensitive part. They have been using certain tools to scan the code of bad practice and static code security vulnerabilities, I'm not sure how far they were able to get. The idea is that it's not as simple as "just clean your code". Just be ready for that.

Reply:
Good thinking, noted 👍